restapi/src/main/java/io/stargate/web/docsapi/service/query/search/db/impl/PathSearchQueryBuilder.java

```diff
@@ -42,6 +44,12 @@ public PathSearchQueryBuilder(List<String> path) {
     this.path = path;
   }
 
+  private List<String> unescape(List<String> escaped) {
```

Comment:
Can we create `DocsApiUtils.convertEscapedCharacters` that accepts a list and then use the utility rather than implement the method here..